app/src/main/java/org/stepic/droid/adaptive/ui/fragments/AdaptiveRatingFragment.kt

```diff
@@ -112,6 +112,15 @@ class AdaptiveRatingFragment: FragmentBase(), AdaptiveRatingView {
         adaptiveRatingPresenter.attachView(this)
     }
 
+    override fun onResume() {
+        super.onResume()
+        analytic
```

Comment:
Won't we receive false events on screen rotation or another events that trigger `onResume`?